src/localization/en.js

```diff
@@ -153,6 +153,9 @@ export default {
     },
 
     currentIncome: {
+      incomeHeader:    'Current Household Income',
```

Comment:
Sorry I missed this before somehow - `title` and `clarifier` seem more useful to me, as opposed to `incomeHeader` and `incomeClarifier`. If you want, we can discuss that more, though it may work better in real-time.